Code/JavaWrappers/gmwrapper/GraphMolJava.i

```diff
@@ -49,7 +68,7 @@
 %include <boost/smart_ptr/shared_array.hpp>
 
 /* Include the base types before anything that will utilize them */
-%include "stdint.i"
+%include "../msvc_stdint.i"
```

Comment:
We can only include this for msvc builds...  we broke the linux java builds.  I'll see if I can fix this.